### PR TITLE
TODO point 2

### PR DESCRIPTION
**Pull request by ashesh2512**
This pull request renames the solution array for the background mesh from `q` to `qcell`, and introduces `qnode` as a nodal solution array for the background mesh. This is in accordance with the tasks listed below - 

https://github.com/jsitaraman/tioga/blob/2b215e11d0cd00bd9b7e3e45efd7bd5184fefeb4/TODO#L36

@jsitaraman  To avoid confusion with the `qnode` that is DG-related, may be it would be better to merge this pull request after the below task has been completed ? 

https://github.com/jsitaraman/tioga/blob/2b215e11d0cd00bd9b7e3e45efd7bd5184fefeb4/TODO#L21

### CODE REVIEW

File: src/tioga.C [hunk 1]
```diff
@@ -212,7 +212,7 @@ void tioga::performConnectivityAMR(void)
   MPI_Barrier(scomm);
   //printf("Finished performConnectivityAMR in %d\n",myid);
   //ierr=0;
-  MPI_Abort(scomm,ierr);
+//  MPI_Abort(scomm,ierr);
```
Comment (sayerhs) [inline src/tioga.C:215]: How is this related to the qnode/qcell change?

Comment (ashesh2512) [reply src/tioga.C]: @sayerhs Thanks for catching this! This should have been a separate pull request. I have reverted that commit.

**Comment by ashesh2512** [comment]
@jsitaraman I have resolved the conflicts resulting from #24.
